src/bindings/python/tests/test_ngraph/test_basic.py

```diff
@@ -4,6 +4,8 @@
 import json
 
 import numpy as np
+from numpy.core.numeric import _ones_like_dispatcher
+from numpy.lib.arraysetops import isin
```

Comment:
Are these two required? It seems like unused imports.

Reply:
Good catch, removed redundant imports.